collector/exec_bsd.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build freebsd dragonfly
+// +build !noexec
+
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type execCollector struct {
+	sysctls []bsdSysctl
+}
+
+func init() {
+	Factories["exec"] = NewExecCollector
+}
+
+// Returns a new Collector exposing system execution statistics
```

Comment:
Please follow the godoc convention and include the function name at the beginning of the sentence. Check with `go lint`.